### PR TITLE
Update EMML151.xml

### PR DESCRIPTION
**Pull request by HagosAbrha**

### CODE REVIEW

File: EMML/1-1000/EMML151.xml [hunk 1]
```diff
@@ -46,8 +46,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                         </support>
                         <extent>
                            <measure unit="leaf">ff 134+1</measure>
-                           <dimensions>
-                              <dim>28 x 20.5 cm</dim>
+                           <dimensions type="outer"unit="mm">
```
Comment (eu-genia) [inline EMML/1-1000/EMML151.xml:49]: you need space before between "outer" and unit

**Review by eu-genia — CHANGES_REQUESTED** [review]
Dear Hagos,
thank you, very good, just one very small correction.
Since you created a new branch I will close the old one.
Eugenia

**Comment by eu-genia** [comment]
Once you have corrected the record in this branch just push the corrected version to this same branch, you do not need a new pull request.

File: EMML/1-1000/EMML151.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                         </support>
                         <extent>
                            <measure unit="leaf">ff 134+1</measure>
-                           <dimensions type="outer"unit="mm">
+                           <dimensions type="outer "unit="mm">
```
Comment (eu-genia) [inline EMML/1-1000/EMML151.xml:49]: sorry, the space should be AFTER the quotation mark, between the two attributes

type="outer" unit="mm"

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by HagosAbrha** [comment]
Ok; thank you!

**Review by eu-genia — APPROVED** [review]
perfect, thank you!

**Comment by eu-genia** [comment]
(this branch is now merged and deleted, so go back to master, fetch+pull, and for new changes create a new branch. thank you!)

**Pull request merged**